style.css

```diff
@@ -0,0 +1,255 @@
+  position: sticky;
+  z-index: 1;
+  top: 0;
+  font-family: "Montserrat", sans-serif;
+}
+
+.menu-btn {
+  background-color: unset;
+  border: none;
+}
+
+.logo {
+  color: white;
+  text-decoration: none;
+}
+
+.headline-container {
+  background-color: #1c1c19;
+  background-image: url(./assets/images/Illustration.png);
+  background-position: right bottom;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  padding: 100px 20px 20px 20px;
+  height: 60vh;
+  gap: 10px;
+  position: relative;
+  font-family: "Montserrat", sans-serif;
+}
+
+.headline-title {
+  color: #ff6b00;
+  font-size: 50px;
+  font-style: unset;
+  font-family: "Crete Round", serif;
+}
+
+.social-media-icones-container {
+  display: flex;
+  list-style: none;
+  margin-left: -3em;
+}
+
+/* today */
+#works-container {
+  display: grid;
+  grid-template-rows: 100px auto 450px 450px 450px 100px;
+}
+
+.recent-works-title-block {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: "Crete Round", serif;
+  font-size: 30px;
+}
+
+.recent-works {
+  display: flex;
+  flex-direction: column;
+  padding: 0 20px 20px 20px;
+  gap: 15px;
+}
+
+.tools-container {
+  display: flex;
+  gap: 20px;
+  margin-left: -2.5em;
+}
+
+.tools {
+  padding: 10px;
+  border: 1px solid #8993a4;
+  list-style: none;
+}
+
+.see-project-btn {
+  background-color: #ff6b00;
+  width: 100px;
+  height: 40px;
+  color: white;
+  border: none;
+}
+
+.see-project-btn:hover {
+  cursor: pointer;
+  border: 2px solid grey;
+}
+
+.see-project-btn:active {
+  scale: 0.9;
+}
+
+.works {
+  display: flex;
+  flex-direction: column;
+  color: white;
+  padding: 120px 20px 20px 20px;
+  background-image: url(./assets/images/bgi2.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  position: relative;
+  gap: 15px;
+}
+
+.works-title {
+  font-family: "Crete Round", serif;
+  font-size: 32px;
+}
+
+.see-project {
+  background-color: #ff6b00;
+  height: 40px;
+  padding: auto;
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  border: none;
+  color: white;
+  transition: ease-in-out;
+}
+
+.see-project:hover {
+  cursor: pointer;
+  border: 2px solid grey;
+}
+
+.see-project:active {
+  scale: 0.95;
+}
+
+/* About Me section */
+#about-me-container {
+  background-color: #1c1a19;
+  background-image: url(./assets/images/Frame.png);
+  background-position: right top 25%;
+  background-repeat: no-repeat;
+  display: grid;
+  grid-template-rows: 72vh 2px 130vh;
+}
+
+.get-resume-btn {
+  width: 10em;
+  height: 40px;
+  background-color: #ff6b00;
+  border: none;
+  color: white;
+}
+
+.get-resume-btn:hover {
+  cursor: pointer;
+  border: 2px solid grey;
+}
+
+.get-resume-btn:active {
+  scale: 0.9;
+}
+
+.about-me-block {
+  display: flex;
+  flex-direction: column;
+  padding: 10vh 20px 4vh 20px;
+  gap: 20px;
+}
+
+.line {
+  height: 2px;
+  width: 90vw;
+  margin: auto;
+  border: none;
+  background-color: gray;
+}
```

Comment:
- Good job for updating the button to stick the Figma  👍 👍 but when you do that you lost your other part. be sure when you change something that does not affect others' part.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/205423031-c1f5514a-c85b-4ecc-8341-23910339aa42.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/205423033-07e41bab-dd73-4b81-b8af-5c2e84045082.png" width="200">|
😍😍